sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -365,6 +365,21 @@ public Void deleteModel(String modelId) {
         return digitalTwinsAsyncClient.deleteModelWithResponse(modelId, context).block();
     }
 
-    //TODO: Decommission Model APIs (waiting for Abhipsa's change to come in)
+    /**
+     * Decommissions a model.
+     * @param modelId The Id of the model to decommission.
+     * @return Void
```

Comment:
No need of this

Reply:
Yeah, javadocs only have the @ return if it returns something other than void. We'll still need them for cases like Mono<Void>, but here you can remove it